Documentation/ceph-csi-drivers.md

```diff
@@ -4,9 +4,13 @@ weight: 32
 indent: true
 ---
 
-# Running ceph CSI drivers with Rook
+# Running Ceph CSI drivers with Rook
 
-Here is a guide on how to implement ceph-csi drivers on a Kubernetes cluster with Rook. The way to implement it should change relatively soon as there is an effort to manage deployments of the ceph-csi plugin within Rook directly ([#1385](https://github.com/rook/rook/issues/1385), [#2059](https://github.com/rook/rook/pull/2059)).
+
+Here is a guide on how to deploy ceph-csi drivers on a Kubernetes cluster with Rook.
+There are two ways to do it: Rook deployed and self serviced.
```

Comment:
What if we only supported the rook automated deployment? I don't think admins want to think about doing it the manual way. What is the need for the self service option?

Reply:
If we can automate it all, would we even need this help topic anymore? Can we make everything happen under the covers?

Reply:
The docs around how to create the storage class and PVCs could be integrated with the other help topics that already exist. Or we may want to keep both CSI and flex options for some time until everyone is on k8s 1.13

Reply:
will refactor